### ISSUE TITLE
Cannot read property 'thumbnail' of undefined

### ISSUE THREAD

**Issue by lokenthapa** [Codestar/codestar-framework#386]
@Codestar  how to fix this bug ...
this bug make edit gallery doesnt show image ..
please help me.
![capture](https://cloud.githubusercontent.com/assets/11645169/17579886/71f031a2-5fb9-11e6-915c-db4821e18093.PNG)

**Comment by NoahjChampion** [comment]
Hi @lokenthapa. 

I think you need to troubleshoot and check for a plugin or theme conflict or update to the latest version (if not) because CF should be working without error for image galleries. 

Please look into this and post more details so someone can help. 

I just tested and everything is working fine without other themes or plugins.

See below. 

<img src="http://g.recordit.co/W7OiTkjxeE.gif">

**Comment by lokenthapa** [comment]
@NoahjChampion 
thank you Noahj for your quick response.. I will do as your instruction...thank you alot...

**Comment by NoahjChampion** [comment]
@lokenthapa welcome. Update when you can. Thanks!

**Comment by upqode** [comment]
Same trouble, but without console error

**Comment by NoahjChampion** [comment]
@upqode 

I don't think you can have the "exact" or "same" issue because an error would be generated in the console. This issue is specifically saying there is an "Uncaught TypeError", which is a Javascript error/conflict, which would or at least should generate a console error. 

First I would check again while performing your action to see if the console generates the error (or doesn't). If the error is not generated then the issue may be with your config or a plugin conflict?

Next, you try deactivating plugins/theme to the default Wordpress setup to see if it is a plugin/theme conflict. 

Last, you can post your config for this @ https://gist.github.com and link me to it so I can try your config and see I get the same issue, so I can look into it. ;) 

Thanks!

**Comment by upqode** [comment]
Can I give access for you to our demo server for lookinkg this bug? in local it works fine.

**Comment by NoahjChampion** [comment]
Sure. Email noahj [at] fearless mode [dot] com.

**Comment by upqode** [comment]
Done, thanks.
